spec/factories/arm.rb

```diff
@@ -43,8 +43,8 @@
 
     trait :with_visit_groups do
       after(:create) do |arm, evaluator|
-        arm.visit_count.times do
-          create(:visit_group, arm: arm)
+        arm.visit_count.times do |n|
+          create(:visit_group, arm: arm, position: n + 1, day: 7 * (n + 1))
```

Comment:
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md - Look into sequence for position attribute